common/djangoapps/student/views/dashboard.py

```diff
@@ -472,6 +473,21 @@ def get_dashboard_course_limit():
     return course_limit
 
 
+@login_required
+@ensure_csrf_cookie
+def send_account_activation_email(request):
```

Comment:
Mobe this endpoint in `openedx/core/djangoapps/user_authn/api` and make it a DRF view.